packages/kyodo-backend/models/RoleAssignment.js

```diff
@@ -0,0 +1,34 @@
+import mongoose, { Schema } from 'mongoose';
+
+const roleAssignmentSchema = new Schema({
+  role: {
+    type: String,
+    enum: ['WORKER'],
+  },
+  operationJSON: String,
+  address: String,
+  accepted: Boolean,
+  taskId: Number,
+});
+
+roleAssignmentSchema.statics.get = async ({ role, taskId }) => {
```

Comment:
Would it be the same if we change this to: 
```
roleAssignmentSchema.statics.get = ({ role, taskId }) => {
  retutn RoleAssignment.findOne({
    taskId,
    role,
  });
};
```
?
P. S. just curious about the pros/cons

Reply:
Good question, I expect it would be same thing in this case?
I'd keep it to be consistent with next static method